cmd/tetragon/main.go

```diff
@@ -115,6 +116,9 @@ func tetragonExecute() error {
 		log.Fatal(err)
 	}
 
+	// enable extra programs/maps loading debug output
+	program.KeepCollection = option.Config.Debug
+
```

Comment:
@olsajiri to get the right log level context, we need something like this:
https://github.com/cilium/tetragon/pull/225/commits/8930368b778f8287f6cc2bde9d92638ede095ce6

Reply:
hm, but this one is just global bool to get the debug messages,
plus option.Config.Debug bool enables debug output in above statement

Reply:
Ah yes, debug can also be set by log-level=debug where option.Config.Debug is just an alias to: log-level=debug... this was done to not break old tetragon helm charts and follow cilium IIRC

So since this touch main tetragon, please check here: https://github.com/cilium/tetragon/blob/main/cmd/tetragon/main.go#L114 we setup logging, so probably you should do something like that helper to get if loglevel is debug or above after we setup that logging... hope it makes sense.

I think we may add a comment to note that option.Config.Debug is just an alias and we should go GetLogLevel() ?

Reply:
> Ah yes, debug can also be set by log-level=debug where option.Config.Debug is just an alias to: log-level=debug... this was done to not break old tetragon helm charts and follow cilium IIRC
> 
> So since this touch main tetragon, please check here: https://github.com/cilium/tetragon/blob/main/cmd/tetragon/main.go#L114 we setup logging, so probably you should do something like that helper to get if loglevel is debug or above after we setup that logging... hope it makes sense.

hum, not sure I follow.. the logger.SetupLogging sets debug level based
on the `option.Config.Debug` value, all this change does is to enable
additional debug output if `option.Config.Debug` value is true

maybe I can make it more explicit? with:

```
        // enable extra programs/maps loading debug output
        if option.Config.Debug {
                program.KeepCollection = true
        }
```

Reply:
ah ok there's --log-level option, I see your point.. how about this then:

```
       // enable extra programs/maps loading debug output
       if logger.DefaultLogger.IsLevelEnabled(logrus.DebugLevel) {
               program.KeepCollection = true
       }
```

Reply:
@olsajiri perfect ;-)  thank you!